dynawo/sources/Modeler/Common/DYNRingBuffer.cpp

```diff
@@ -103,4 +111,14 @@ RingBuffer::interpol(const std::pair<double, double>& p1, const std::pair<double
   return a * time + b;
 }
 
+std::vector<std::pair<double, double>>
+RingBuffer::points() const {
```

Comment:
I generally prefer to pass the vector as reference parameter to avoid useless copies

Reply:
when upgrade to c++11 will be done, it will be a move, not a copy, so I'm not sure this is relevant

Reply:
upgrade to c++11 is next year at best, so in the meantime it would preferable IMHO

Reply:
OK